packages/ra-core/src/routing/useCreatePath.spec.tsx

```diff
@@ -50,7 +51,11 @@ describe('useCreatePath', () => {
     });
 
     it('creates valid links when used without a basename', async () => {
-        render(<AtRoot />);
+        render(
+            <TestMemoryRouter>
```

Comment:
I wonder why you removed the memory router wrapping in the story to add it back here. Would you care to explain?

Reply:
It's because the story previously relied on `FakeBrowserDecorator` to provide a memory/hash history, and was autonomous to create a new router if  `FakeBrowserDecorator` was absent. Now, `FakeBrowserDecorator` provides its own router, but the story in standalone use no longer does it, so we need to provide the router in the tests.

Hopefully my explanations are clear enough :sweat_smile: